### DISCUSSION TITLE
Elgato HDX 120 modes

### DISCUSSION THREAD

**Discussion by d4rkdpg** [awawa-dev/HyperHDR#485] [General]
Hi I recently replaced my MS2109 with elgato hd60x.  Im trying to list supported formats for this device like the hyper hdr blog review for this device but cannot see 1080p 120fps.  Tried on ubuntu (dell laptop) and raspberrypi os  (raspberry pi 4) I'm running this:

v4l2-ctl --list-formats-ext -d 2
ioctl: VIDIOC_ENUM_FMT
	Type: Video Capture

	[0]: 'YUYV' (YUYV 4:2:2)
		Size: Discrete 1920x1080
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 1600x1200
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 1280x720
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 720x576
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 720x480
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 640x480
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
	[1]: 'NV12' (Y/CbCr 4:2:0)
		Size: Discrete 3840x2160
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 2560x1440
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 1920x1080
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 1600x1200
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 1280x720
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 720x576
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 720x480
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
		Size: Discrete 640x480
			Interval: Discrete 0.017s (60.000 fps)
			Interval: Discrete 0.020s (50.000 fps)
			Interval: Discrete 0.033s (30.000 fps)
			Interval: Discrete 0.040s (25.000 fps)
	[2]: '' (30313050-0000-0010-8000-00aa003)

The firmware is up to date as of yesterday.  How can I achieve this?

Thanks

**Comment by awawa-dev** [comment]
Hi
It works for me under the Windows 10 because I use that system to test the grabber's latency. Linux drivers are behind this grabber capability.

**Reply by d4rkdpg** [reply]
I see ok thanks any unofficial drivers or workarounds ?

**Reply by awawa-dev** [reply]
I haven't found any. It's not the first such situation with USB grabbers under Linux e.g. popular MS2109 doesn't support NV12 codec which Windows driver supports.

**Comment by d4rkdpg** [comment]
Just to add I tested this on windows and can get 120 in 4 k capture utility but with hyper hdr still shows up max 60 fps, 120 not shown any ideas?

**Reply by awawa-dev** [reply]
Did you connect to USB3.0 and use 1080p signal on the input?

**Reply by d4rkdpg** [reply]
Yes i did

**Reply by awawa-dev** [reply]
1920x1080x120 is only supported with NV12 codec.

**Reply by d4rkdpg** [reply]
Yes I know and on my Windows installation of hyper hdr I cannot see 120 fps in video capture section under 1080p NV12.  But under elgato utitlty under capture 120 input no problem.  I’m I missing something here ?

**Reply by awawa-dev** [reply]
Verify your HyperHDR logs. Video modes are listed there.
